packages/model-viewer/src/test/features/scene-graph/nodes/primitive-node-spec.ts

```diff
@@ -119,12 +121,26 @@ suite('scene-graph/model/mesh-primitives', () => {
       materials = new Array<MeshStandardMaterial>();
       for (const primitive of primitives) {
         materials.push(
-            primitive.enableVariant('Purple Yellow') as MeshStandardMaterial);
+            await primitive.enableVariant('Purple Yellow') as
+            MeshStandardMaterial);
       }
       expect(materials.find((material: MeshStandardMaterial) => {
         return material.name === 'purple';
       })).to.not.be.null;
     });
+
+    test('Primitive switches to default material', async () => {
```

Comment:
This is good, but it's not a public method. Let's also include a more end-to-end version that sets it via the public `variantName` API. And let's also update the documentation on that.

Reply:
Added a test using the `variantName` API, a bit tricky to verify without being able to access a gltf-primitive mapping on the model, have a look, updating the docs next...